plugins/dispatch/src/main/java/org/qbicc/plugin/dispatch/DispatchTables.java

```diff
@@ -261,13 +260,13 @@ void emitVTable(LoadedTypeDefinition cls) {
                 MethodElement stub = methodFinder.getMethod(vtable[i].isAbstract() ? "raiseAbstractMethodError" : "raiseUnsatisfiedLinkErrorDispatchStub");
                 Function stubImpl = ctxt.getExactFunction(stub);
                 FunctionDeclaration decl = programModule.declareFunction(stub, stubImpl.getName(), stubImpl.getValueType());
-                PointerLiteral literal = ctxt.getLiteralFactory().literalOf(decl.getPointer());
+                Literal literal = ctxt.getLiteralFactory().literalOf(decl.getPointer());
```

Comment:
Do we still need `decl.getPointer()` here (and in similar cases)?  If I'm following the flow in LiteralFactory correctly `literalOf(decl)` and `literalOf(decl.getPointer())` will both end up returning equivalent `InstanceMethodLiteral` instances